### PR TITLE
[ALLUXIO-2165] add LOG.debug("Check if {} is a file", path) into  SwiftUnderFileSyst…

### PR DESCRIPTION
**Pull request by EagleAuburn**
add LOG.debug("Check if {} is a file", path) into  SwiftUnderFileSystem::isFile
Jira ticket: https://alluxio.atlassian.net/browse/ALLUXIO-2165?filter=-1

### CODE REVIEW

**Comment by aaudiber** [comment]
jenkins, ok to test

**Comment by aaudiber** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
@EagleAuburn, it looks like your commits on GitHub aren't linked to a valid Github account yet. For us to properly credit your contributions, could you please set up your Github account by following the Github instructions [here](https://help.github.com/articles/setting-your-email-in-git)?

**Comment by EagleAuburn** [comment]
I think I have already linked to a valid Github account. I checked, all shows correct information

**Comment by aaudiber** [comment]
@EagleAuburn The commits are made under the email address yuanqi.chen@equifax.com, but it looks like your github account is linked to chenyuanqi2012@gmail.com instead. You could add yuanqi.chen@equifax.com as a github email address, or change the commits to use chenyuanqi2012@gmail.com.

**Comment by EagleAuburn** [comment]
Thanks, Have been updated.

File: underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java [hunk 1]
```diff
@@ -297,6 +297,7 @@ public long getSpace(String path, SpaceType type) throws IOException {
 
   @Override
   public boolean isFile(String path) throws IOException {
+    LOG.debug("Check if {} is a file", path);
```
Comment (calvinjia) [inline underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java]: To be consistent with the other comment, could you change this to `Is file {}`?

**Comment by aaudiber** [comment]
@EagleAuburn hm, the commits are still linked to equifax. This article might be helpful: https://www.git-tower.com/learn/git/faq/change-author-name-email

File: underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java (hunk 1 above)
Comment (calvinjia) [inline underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java]: @EagleAuburn ping

Comment (EagleAuburn) [inline underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java]: Will updated it, Thanks

**Comment by EagleAuburn** [comment]
I have already updated the user.email and confirmed it. Could you please help to double check it. Thanks!

**Comment by aaudiber** [comment]
When I check out your branch with

`git fetch upstream pull/3798/head && git checkout FETCH_HEAD`

and then run `git log`, I see

image

so it looks like your commits are definitely still associated with `yuanqi.chen@equifax.com`

You can fix the email address by running the following:

**Checkout your commits**
1. `git fetch upstream pull/3798/head && git checkout FETCH_HEAD`
**Edit your commits**
2. `git rebase -i HEAD~2`
3. change the "pick" to "e" on the first two lines and save
**Fix the email on the first commit**
4. `git commit --amend --author "Yuanqi Chen <chenyuanqi2012@gmail.com>"` and save
5. `git rebase --continue`
**Fix the email on the second commit**
6. `git commit --amend --author "Yuanqi Chen <chenyuanqi2012@gmail.com>"` and save
7. `git rebase --continue`
**Push the commits with the fixed emails back to this branch (-f is needed since we overwrote history)**
8. `git push -f origin HEAD:refs/heads/alluxio-2165-add-log-debug-swiftUnderFileSystem`

**Comment by aaudiber** [comment]
@EagleAuburn

**Comment by calvinjia** [comment]
@EagleAuburn Any updates on this?

**Comment by aaudiber** [comment]
Closing due to inactivity, feel free to reopen if you have time to return to this @EagleAuburn

**Pull request closed without merging**